libraries/vircadia-client/src/common.h

```diff
@@ -0,0 +1,24 @@
+//
+//  common.h
+//  libraries/client/src
+//
+//  Created by Nshan G. on 1 March 2022.
+//  Copyright 2022 Vircadia contributors.
+//
```

Comment:
```suggestion
//  Copyright 2022 Vircadia contributors.
//
//  Distributed under the Apache License, Version 2.0.
//  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
//
```
Missing license lines, here and elsewhere.